data/soapvalidator/Mail/Attachments/Message-Send-WAttach-WScanEnabled.xml

```diff
@@ -0,0 +1,313 @@
+            <t:response>
+                <t:select path="//admin:AuthResponse/admin:authToken" set="authToken"/>
+            </t:response>
+        </t:test>
+
+        <t:test required="true" depends="admin_login">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account1.name}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]'
+                          set="test_acct1.server"/>
+            </t:response>
+        </t:test>
+
+        <t:test required="true" depends="admin_login">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account2.name}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+        </t:test>
+
+        <t:property name="server.zimbraAccount" value="${test_acct1.server}"/>
+
+
+        <t:test id="getallserversrequest01" depends="admin_login">
+            <t:request>
+                <GetAllServersRequest xmlns="urn:zimbraAdmin"/>
+            </t:request>
+            <t:response>
+                <t:select
+                        path='//admin:GetAllServersResponse/admin:server[admin:a[@n="zimbraServiceEnabled"]="mailbox"][1]'
+                        attr="id" set="server.id1"/>
+            </t:response>
+        </t:test>
+
+        <t:test id="ModifyServer01" depends="admin_login">
+            <t:request>
+                <ModifyServerRequest xmlns="urn:zimbraAdmin">
+                    <id>${server.id1}</id>
+                    <a n="zimbraClamAVBindAddress">${zimbra.clamav.bind.address}</a>
+                </ModifyServerRequest>
+            </t:request>
+        </t:test>
+
+        <t:test id="ModifyServer02" depends="admin_login">
+            <t:request>
+                <ModifyServerRequest xmlns="urn:zimbraAdmin">
+                    <id>${server.id1}</id>
+                    <a n="zimbraAttachmentsScanURL">${zimbra.attachment.scan.url}</a>
+                </ModifyServerRequest>
+            </t:request>
+        </t:test>
+
+        <t:test id="ModifyServer03" depends="admin_login">
+            <t:request>
+                <ModifyServerRequest xmlns="urn:zimbraAdmin">
+                    <id>${server.id1}</id>
+                    <a n="zimbraAttachmentsScanEnabled">${zimbra.attachment.scan.enabled}</a>
+                </ModifyServerRequest>
+            </t:request>
+        </t:test>
+
+        <t:staftask>
+            <t:request>
+                <server>${zimbraServer.name}</server>
+                <service>PROCESS</service>
+                <params>START SHELL COMMAND "su - zimbra -c \'/opt/zimbra/bin/zmcontrol restart\'" RETURNSTDOUT
```

Comment:
Can we avoid doing zmcontrol restart and instead start only service to make the changes effective?

Reply:
For parallelism, zmcontrol restart should be separated in different tests (should run such tests stand alone one by one). This interrupts and fails other tests.

Reply:
zmmailboxd restart should be avoided too, in short environment should be intact when running multiple tests.

Reply:
But it's required here.. without this how localconfig/ldap-attributes would take effect? Please assist if there is a other better way of doing this.

Reply:
Agree. I'm not saying to remove anything which is required, just suggesting to create some special group in the test so it won't run together. For e.g. restart-server.

Reply:
hmm. like this https://github.com/Zimbra/zm-soap-harness/pull/172#discussion_r1036913341.
Thank you let me try if it works with the zmmailbox restart.

Reply:
Separate folder is not required, test should be identified by test group for e.g.
smoke
bhr
sanity
functional
restart-server

Server QA folks should comment in this thread who are using SOAP automation framework.

Reply:
Hi Rakesh More,
You need to specify for test cases SendMsgRequest_wAttachment_01 and SendMsgRequest_wAttachment_02 - instead of type="bhr" please specify type "restart-server"

Reply:
Let's merge this now and we will circle this back